uptane-standard.md

```diff
@@ -340,6 +341,8 @@ Attackers seeking to interfere with the functionality of vehicle ECUs in order t
 * *Rollback attack:* Cause an ECU to install a previously valid software revision that is older than the currently installed version.
 * *Endless data attack:* Send a large amount of data to an ECU until it runs out of storage, possibly causing the ECU to fail to operate.
 * *Mix-and-match attack:* Install a malicious software bundle in which some of the updates do not interoperate properly. This could be accomplished even if all of the individual images being installed are valid, as long as valid versions exist that are mutually incompatible.
+* *Arbitrary command injection:* Sending commands through exposed services or unsecure protocols of an ECU to perform malicious actions or cause it to fail.
+* *Arbitrary data injection:* Sending data into communication lines within the vehicle, potentially to spoof as another ECU or to tamper existing data in transit.
```

Comment:
I think we're not blocking these categorically.  Uptane will stopping Arbitrary software from being loaded by SOTA, but the ECU software loaded by the OEM could technically do whatever it wants.

Reply:
I agree now.
In that case it may be something due to one of the "Attacker goals" being "Control ECUs within the vehicle, and possible the vehicle itself"

If the threat model does not consider things outside the OTA, such as in-vehicle communications, then perhaps we would either remove or reword to the following:
"Control ECUs within the vehicle, and possible the vehicle itself, through malicious SOTA installation"

Reply:
Okay, let's add that to the meeting agenda as well.  I'm fine with this clarification.

Reply:
At the 5-9-23 meeting, it was decided rather than adding the lines mentioned above, that we should amend the bullet at line #306, under attacker goals, to “Control ECUs within the vehicle, and possibly the vehicle itself, through malicious SOTA installation”.